docs/source/gbd2017_models/causes/covid_19/index.rst

```diff
@@ -11,35 +11,285 @@ Covid-19 Custom SEIR Vivarium Cause Model
+
+.. math:: n_\text{simulants to sample} = R_0 * (\gamma_1 + \gamma_2) * duration_{\text{time step}}
+
+.. important::
+
+	If :math:`R_t` from the IHME forecast model is being used as the simulation :math:`R_0` value, simulants should only be drawn from the pool of simulants who were initialized into the population in the :math:`S` state. Notably, if a simulant is initialized into the :math:`S` state and later progresses to later model states, they are still eligible to be sampled.
+
+Then, if a sampled simulant's current model state is :math:`S`, that simulant should be transitioned to model state :math:`E`. If a sampled simulant's current model state is not :math:`E`, no change should be made.
+
+.. note::
+
+	This method can be expanded to include correlated mixing structures so that certain simulants are more likely to "mix" with others on the basis of age, location, and/or disease state.
+
+	Any correlated mixing structure will need to be expanded upon to instruct how to sample simulants based on these variables. 
+
+Duration Based Transitions
+""""""""""""""""""""""""""
+
+The transistions from model states :math:`E` --> :math:`I_1`, :math:`I_1` --> :math:`I_2`, :math:`I_2` --> :math:`R` will be modeled as duration-based transitions. Notably, the duration of each of these states will be dependent on whether a simulant has been determined to die due to covid-19 at the onset of their infection (as described in the Mortality_ section).
+
+The following table defines the state durations before a simulant will transition to the next state given that they were determined **not** to die due to covid-19:
+
+.. list-table:: State Durations
+   :widths: 5 5 10
+   :header-rows: 1
+
+   * - Source State
+     - Duration
+     - Note
+   * - :math:`E`
+     - :math:`1/\sigma`
+     - :math:`\sigma` obtained from IHME forecast model
+   * - :math:`I_1`
+     - :math:`1/\gamma_1`
+     - :math:`\gamma_1` obtained from IHME forecast model
+   * - :math:`I_2`
+     - :math:`1/\gamma_2`
+     - :math:`\gamma_2` obtained from IHME forecast model
+
+If it **was** determined that a simulant will die due to covid-19 (as described in the Mortality_ section), assume that the individual dies a the duration of the :math:`1_2` model state instead of transitioning them to the :math:`R` state.
+
+.. warning::
+
+	Determine if these values from IHME forecast model include cases of mortality and update methodology accordingly
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+There are several assumptions used in this cause model:
+
+- Recovered individuals are no longer susceptible to covid-19
+
+- There is no birth prevalence of covid-19
+
+- These is random mixing of the population (unless :math:`\alpha` parameter is not equal to one or correlated mixing structure is introduced)
+
+- The transmission probability is homogenous across demographic groups
```

Comment:
How hard would it be to model sub-populations?

Reply:
For the UW model (which will be in the upcoming PR), we are considering modeling the UW population explicitly and then allowing for some sort of constant covid-19 flow into the model from the surrounding population. Not quite a subpopulation, but gets at the concept that our model population is a subpopulation of the larger world!

I think that modeling subpopulations like dorms and classrooms would be really cool, but it's beyond our initial scope from my discussions with abie.